package.json

```diff
@@ -48,6 +48,11 @@
                 "when": "editorTextFocus"
             },
             {
+                "key": "ctrl+[",
+                "command": "extension.vim_esc",
```

Comment:
So, something like this?

package.json:
```
{
  "key": "ctrl+[",
  "command": "extension.ctrl+[",
  "when": "editorTextFocus"
}
```

actions.ts (I think that's the file name):
```
CommandControlOpenBracket extends CommandEscape {
  keys = ["ctrl+["]
}
```

I don't know Typescript, but it seems like I should be able to leverage the existing escape command.